File: PkmWebServer/Models/States/MoveStateModel.cs
namespace PkmWebServer.Models.States;
public class MoveStateModel
{
    public int MoveId { get; set; }
    public bool IsAnswered { get; set; } = false;
    public int Points { get; set; }
    public HintStateModel DamageClass { get; set; } = new();
    public HintStateModel Type { get; set; } = new();
    public HintStateModel Stats { get; set; } = new();
    public HintStateModel FlavorText { get; set; } = new();
}

File: PkmWeb/PkmWeb/src/app/guesser/guess/move-state.model.ts
import { HintState } from "@guesser/hint/models";

export interface MoveState {
  id: number,
  name: string,
  levelLearned: number,
  isAnswered: boolean,
  points: number,
  damageClass: HintState,
  type: HintState,
  stats: HintState,
  flavorText: HintState
}
